xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -1087,7 +1087,8 @@ handler.ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousA
 
 ## Remarks
  Set this property to `true` when requests made by the <xref:System.Net.Http.HttpClientHandler> object should, if requested by the server, be authenticated using the credentials of the currently logged on user. For client applications, this is the desired behavior in most scenarios. For middle-tier applications, such as ASP.NET applications, instead of using this property, you would typically set the <xref:System.Net.Http.HttpClientHandler.Credentials%2A> property to the credentials of the client on whose behalf the request is made.
-
+ 
+ This property doesn't affect proxy credentials. Proxy credentials need to be explicitly set by the <xref:System.Net.Http.HttpClientHandler.DefaultProxyCredentials> property. 
```

Comment:
>"Proxy credentials need to be explicitly set by the <xref:System.Net.Http.HttpClientHandler.DefaultProxyCredentials> property."

The HttpClientHandler.DefaultProxyCredentials property only provides credentials when a system-default proxy is used, i.e. one that has been discovered via operating system configuration such as environment variables or system proxy settings.

But if the HttpClientHandler.Proxy is set to a non-null value, then credentials for a proxy would come from that object via its [IWebProxy.Credentials](https://docs.microsoft.com/en-us/dotnet/api/system.net.iwebproxy.credentials?view=netcore-3.1#System_Net_IWebProxy_Credentials) property.

So, you should expand upon your text above.